src/pages/Statistics/Statistics.tsx

```diff
@@ -13,21 +31,313 @@ export default function Statistics() {
   const [startDate, setStartDate] = useState(location?.state?.startDate ?? "");
   const [endDate, setEndDate] = useState(location?.state?.endDate ?? "");
 
+  const settingsNames = user!.settings.map((e) => e.key);
+  const [radioValue, setRadioValue] = useState(2);
+  const [graphTitle, setGraphTitle] = useState("Weekly Happiness");
+  const [graphSubTitle, setGraphSubTitle] = useState("");
+  const [viewingEntry, setViewingEntry] = useState<Happiness | undefined>(
+    undefined,
+  );
+  const [calCollapsed, setCalCollapsed] = useState<boolean>(false);
+  const [start, setStart] = useState(
+    new Date(
+      new Date().getFullYear(),
+      new Date().getMonth(),
+      new Date().getDate() - 7,
+    ),
+  );
+  const [end, setEnd] = useState(
+    new Date(
+      new Date().getFullYear(),
+      new Date().getMonth(),
+      new Date().getDate(),
+    ),
+  );
+
+  useEffect(() => {
+    // @ts-ignore
+    window.HSOverlay.open(document.querySelector("#show-happiness-modal"));
+  }, [viewingEntry]);
+
+  const { width, height } = useWindowDimensions();
+
+  const {
+    isLoading,
+    data,
+    isError,
+    refetch,
+  }: {
+    isLoading: boolean;
+    data: Happiness[] | undefined;
+    isError: boolean;
+    refetch: (
+      queryFnArgs?: undefined,
+    ) => Promise<Happiness[] | undefined | unknown>;
+  } = useQuery({
+    queryKey: QueryKeys.FETCH_HAPPINESS + " graph query",
```

Comment:
```suggestion
    queryKey: [QueryKeys.FETCH_HAPPINESS, " graph query", {start: start}, {end: end}]
```
You need to make sure query keys are properly formatted and include any parameters that impacts the data that they retrieve.